users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -49,6 +49,12 @@ def add_specaug_layer(
                 freq_mask_max_proportion (float): The maximum proportion of the frequency axis that can be masked. Default is 1.
                 max_time_num_seq_len_divisor (int):
                     The divisor for the sequence length to determine the maximum number of time masks. Default is 0.7.
+                mask_based_on_variance (bool): Whether to mask based on variance or not. Default is False.
+                variance_factor (float): The factor that determines the probability of masking a feature based on variance. Default is 0.5.
+                max_number_masks_for_variance_specaug (int): The total maximum number of masks to be applied if variance-based masking is used. Default is 50.
+                variance_mask_schedule (Dict[int, float]):
```

Comment:
This all still have variance in their name. As I said, I would prefer something like "filter-based".